src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -180,7 +180,10 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
   const updateVenueRoom = useCallback(async () => {
     if (!user || !roomVenueId) return;
 
-    const embedUrl = convertToEmbeddableUrl({ url: venueValues.iframeUrl });
+    const embedUrl = convertToEmbeddableUrl({
```

Comment:
[NOTE]
`convertToEmbeddableUrl()` should probably be used inside `updateVenueNG()` to make sure every call to it has the URL updated based on `autoPlay` value that's been given to it, maybe even save raw and updated URL